java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -4410,6 +4410,23 @@ void testGetMapValue() {
       assertColumnsAreEqual(expected, res);
     }
   }
+    @Test
+    void testGetMapKeyExistence() {
+        List<HostColumnVector.StructData> list1 = Arrays.asList(new HostColumnVector.StructData(Arrays.asList("a", "b")));
+        List<HostColumnVector.StructData> list2 = Arrays.asList(new HostColumnVector.StructData(Arrays.asList("a", "c")));
+        List<HostColumnVector.StructData> list3 = Arrays.asList(new HostColumnVector.StructData(Arrays.asList("e", "d")));
+        List<HostColumnVector.StructData> list4 = Arrays.asList(new HostColumnVector.StructData(Arrays.asList("a", "g")));
+        List<HostColumnVector.StructData> list5 = Arrays.asList(new HostColumnVector.StructData(Arrays.asList("f", "h")));
+        List<HostColumnVector.StructData> list6 = Arrays.asList(new HostColumnVector.StructData(Arrays.asList("a", null)));
+        List<HostColumnVector.StructData> list7 = Arrays.asList(new HostColumnVector.StructData(Arrays.asList(null, null)));
```

Comment:
The result column exposes the offset details of the internal child column, personally it would be better to convert to a bool column by `resultCol.notEq(Scalar.fromInt(-1))`, then `false ` means not found.

Reply:
Why should it not pass ?